src/org/opensolaris/opengrok/web/api/v1/RestApp.java

```diff
@@ -30,7 +30,8 @@
 public class RestApp extends ResourceConfig {
 
     public RestApp() {
-        packages("org.opensolaris.opengrok.web.api.v1.controller", "org.opensolaris.opengrok.web.api.v1.filter");
+        packages("org.opensolaris.opengrok.web.api.constraints", "org.opensolaris.opengrok.web.api.error");
+        packages(true, "org.opensolaris.opengrok.web.api.v1");
```

Comment:
`true` means that the package should be traversed recursively

Reply:
these two lines wouldn't work as expected?
```java
packages("org.opensolaris.opengrok.web.api.v1.controller", "org.opensolaris.opengrok.web.api.v1.filter");
packages("org.opensolaris.opengrok.web.api.constraints", "org.opensolaris.opengrok.web.api.error");
```

But I'm ok with this, just askin.

Reply:
Yeah, your solution would work as well. If it's done this way then there is no need to modify `RestApp` if some additional package would be created under `org.opensolaris.opengrok.web.api.v1` package.

Reply:
fine, ok.